M6-Backend/MockFacility.test.js

```diff
@@ -0,0 +1,210 @@
+//Test Set 1 --> Testing the interface specific for module facility
+const interfacespecific = require('./interfacespecific');
+
+test("interfacespecific invalid input", ()=>{
+  expect(interfacespecific("ll@@@@}}")).not.toEqual('posts');
+})
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/HizanLi/CPEN321/pullRequest?prid=9816566)
```suggestion
});
```